assets/global-classes.less

```diff
@@ -86,4 +95,33 @@
   background-color: white;
   border: 1px solid @tangerine;
   border-radius: 4px;
+}
+
+
+// -------------- FORMS & INPUT ------------------
+
+.form-title {
+  font-size: 18px;
+  display: block;
+  margin-bottom: 1em;
+}
+
+input[type="text"], input[type="password"] {
+  border: 2px solid #888888;
+  border-radius: 2px;
+  font-size: 16px;
+  padding: .5em;
+  margin: 10px 0 10px 0;
+}
+
+button {
```

Comment:
can you change this to a class and name it "btn-primary" or "btn-secondary" or something. We don't necessarily want all buttons on the website to have all of these properties.